internal/util/update/resource-merge.go

```diff
@@ -92,8 +92,6 @@ func (u ResourceMergeUpdater) Update(options UpdateOptions) error {
 		OriginalPath: original.AbsPath(),
 		UpdatedPath:  updated.AbsPath(),
 		DestPath:     options.PackagePath,
-		// TODO: Write a test to ensure this is set
-		MergeOnPath: true,
```

Comment:
What are the implications for this ? @mortent

Reply:
This is safe. The interface for Merge3 has changed a bit, but `MergeOnPath` defaults to `true`.